cypress/tests/tests/rwa-login-test.spec.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+const username = "Abel1";
+const password = "abel1";
```

Comment:
read about arrays. instead of 2 consts you can use array with user data

```suggestion
const testUser = {
  firstName: "Abel",
  lastName: "Fernandez",
  username: "Abel1",
  password: "abel1"
}
```